server/ai_mediaserver.go

```diff
@@ -360,3 +361,26 @@ func (ls *LivepeerServer) ImageToVideoResult() http.Handler {
 		_ = json.NewEncoder(w).Encode(resp)
 	})
 }
+
+func (ls *LivepeerServer) RealtimeStart() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		streamName := r.FormValue("stream")
+		if streamName == "" {
+			http.Error(w, "Missing stream name", http.StatusBadRequest)
+			return
+		}
+		requestID := string(core.RandomManifestID())
+		params := aiRequestParams{
+			node:        ls.LivepeerNode,
+			os:          drivers.NodeStorage.NewSession(requestID),
+			sessManager: ls.AISessionManager,
+		}
+		ctx := clog.AddVal(r.Context(), "request_id", requestID)
+		clog.Infof(ctx, "Received realtime AI request")
+		req := worker.StartRealtimeToRealtimeFormdataRequestBody{
+			// TODO set model and initial parameters here if necessary (eg, prompt)
+		}
+		resp, err := processAIRequest(ctx, params, req)
```

Comment:
I don't think we should follow this AI flow here. I guess what we're trying to do is more similar to transcoding.

Reply:
Why not? This selects an orchestrator and allows us to receive and process a response from it (so we know where to send and receive media from)

This method works for now - if you want to change it later as part of the selection work that is fine but this gets us going with the existing RPC mechanisms

Reply:
I think I see what @leszko is saying, up until now we have just been sending in synchronous AI jobs, the pipeline runs and returns the result. Whereas here the request is to start up the realtime to realtime stream basically? That said I don't think it's huge issue when there's lots of common code that it make sense to reuse by following the same flow.

Reply:
Yes, I think we can "re-use" it for this starting of AI runner. But then, the next question would be, when does the runner container stops? Or who stops it? We don't need to solve it in this PR, but something to consider.

Reply:
Yep as @mjh1 says this basically makes a synchronous request using the normal AI inference path. This is pretty convenient, because it gives us an entry point to kick off the rest of the process - see https://github.com/livepeer/go-livepeer/pull/3211 for the beginnings of that.

I expect that we'd be building things like our job monitoring / re-selection, payments, etc somewhere near this '[main loop](https://github.com/livepeer/go-livepeer/pull/3211/files#diff-5dd7b71c3614e76641eba97465018e0d92a2bf6be8918853ac85763e4c2010c6)'

> But then, the next question would be, when does the runner container stops? Or who stops it? We don't need to solve it in this PR, but something to consider.

This is would be a basic RPC request which can be done via whatever call-path makes sense. However, as far as job tear-down goes, I am not sure if we can rely on signaling for that (anything can disappear without notice), so we should ensure things still work well in the absence of explicit tear-down.

We'll need to solve a similar issue with calls to the model control API. The gateway will need some handle to the orchestrator session, the orchestrator to the (remote) worker, and possibly the worker to the runner (if runners have persistent job state). @victorges for visibility since this is his area

Reply:
> This is would be a basic RPC request which can be done via whatever call-path makes sense. However, as far as job tear-down goes, I am not sure if we can rely on signaling for that (anything can disappear without notice), so we should ensure things still work well in the absence of explicit tear-down.

Yeah, the best approach from the Distributed (and Decentralized) system perspective is what we do for Transcoding:
1. The params are included in the stream (segments)
2. The AI (Transcoding) pipelines is initialized when the first segment comes in
3. The tear down is down on some timeout (so when there are no segments coming for x min, then it tears down)

Any signaling of warm up / tear down is an "optimization" on top of that. This approach is good for reliability, because a lot may happen, O may go down, we may swap the orchestrator, G may go down (before signaling something), etc. 

CC: @victorges 

Again, I think we can move this discussion outside this PR. The PR can be merged and we can discuss it separately.